File: Python/uberBlack.py
'''
Created Date: Friday July 30th 2021 11:27:55 pm
Author: Andrés X. Vargas
-----
Last Modified: Friday July 30th 2021 11:30:04 pm
Modified By: the developer known as Andrés X. Vargas at <axvargas@fiec.espol.edu.ec>
-----
Copyright (c) 2021 MattuApps
'''
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: JS/UberBlack.js
/*
 * Created Date: Friday July 30th 2021 11:49:13 pm
 * Author: Andrés X. Vargas
 * -----
 * Last Modified: Friday July 30th 2021 11:52:38 pm
 * Modified By: the developer known as Andrés X. Vargas at <axvargas@fiec.espol.edu.ec>
 * -----
 * Copyright (c) 2021 MattuApps
 */
class UberBlack extends Car{
    constructor(license, driver, typeCarAccepted, seatsMaterial) {
        super(license, driver)
        this.typeCarAccepted = typeCarAccepted
        this.seatsMaterial = seatsMaterial
    }
}